### PR TITLE
feat: add reddit into social share

### PR DESCRIPTION
**Pull request by megatheikal**

### CODE REVIEW

File: components/Lego/SocialShare/Reddit.vue [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    `http://www.reddit.com/submit?url=${encodeURI(
+      path?.value ?? ""
+    )}&title=${encodeURI(props.text ?? "")}`
+);
+</script>
+
+<template>
+  <NuxtLink :to="url" target="_blank">
+    <slot>
+      <Icon name="uil:reddit-alien-alt" />
```
Comment (zernonia) [inline components/Lego/SocialShare/Reddit.vue:18]: can we change this to `mdi:reddit`? `uil:reddit-alien-alt` doesn't looks so nice

Comment (megatheikal) [reply components/Lego/SocialShare/Reddit.vue:18]: okay sure

Comment (zernonia) [reply components/Lego/SocialShare/Reddit.vue]: Awesome! Thanks @megatheikal

**Comment by zernonia** [comment]
Thanks for the PR @megatheikal ! Really appreciate the contribution! 🙌🏼

**Pull request merged**